documentation/book/ref-storage-jbod.adoc

```diff
@@ -0,0 +1,43 @@
+// Module included in the following assemblies:
+//
+// assembly-storage.adoc
+
+[id='ref-jbod-storage-{context}']
+= JBOD storage
+
+The JBOD storage allows you to provide more volumes to the Kafka brokers to store data.
+As well as providing more storage, using multiple disks can improve performance.
+
+To use JBOD storage, the `type` has to be set to `jbod`.
+A JBOD storage is described by one or more volumes and each of them can be an xref:ref-ephemeral-storage-{context}[Ephemeral] or a xref:ref-persistent-storage-{context}[Persistent] one.
+Each volume declaration follows the same rules and constraints as for Ephemeral and Persistent storage.
+For example, it is not possible to change the size for a Persistent storage when it is already provisioned.
+The JBOD storage provides a way for putting together more of them.
+
+Using the `volumes` property, it is possible to declare the disks which make the JBOD storage.
+
+.Example fragment of JBOD storage configuration
+[source,yaml]
+----
+# ...
+storage:
+  type: jbod
+  volumes:
+  - id: 0
+    type: persistent-claim
+    size: 100Gi
+    deleteClaim: false
+  - id: 1
+    type: persistent-claim
+    size: 100Gi
+    deleteClaim: false
+# ...
+----
+
+It is not possible to change the `id` property for a volume.
```

Comment:
This statement seems a little ambiguous. Did you mean the following: "It is not possible to change the `id` property for a volume after it is provisioned"?

Do the `id` properties need to be set to 0, 1, 2, 3, 4 etc. in ascending order?

Reply:
Yes, to both the questions :-)

Reply:
Thanks. I just need to update that sentence before the PR is merged.